src/bootstrap/bootstrap.py

```diff
@@ -1004,6 +1004,16 @@ def bootstrap(help_triggered):
         with open(toml_path) as config:
             build.config_toml = config.read()
 
+    profile = build.get_toml('profile')
+    if profile is not None:
+        include_file = 'config.{}.toml'.format(profile)
+        include_dir = os.path.join(build.rust_root, 'src', 'bootstrap', 'defaults')
+        include_path = os.path.join(include_dir, include_file)
+        # HACK: This works because `build.get_toml()` returns the first match it finds for a
+        # specific key, so appending our defaults at the end allows the user to override them
+        with open(include_path) as included_toml:
+            build.config_toml += os.linesep + included_toml.read()
```

Comment:
Sorry, replied in the other thread, copy pasting the reply here:

Running this:
```
./x.py setup compiler
./x.py build
```

was throwing this error:

```
Updating only changed submodules
Submodules updated in 0.02 seconds
    Finished dev [unoptimized + debuginfo] target(s) in 0.11s


failed to execute command: "/home/alarsyo/work/rust/build/x86_64-unknown-linux-gnu/ci-llvm/bin/llvm-config" "--libfiles"
error: No such file or directory (os error 2)


failed to run: /home/alarsyo/work/rust/build/bootstrap/debug/bootstrap build
Build completed unsuccessfully in 0:00:00
```

Because bootstrap.py is responsible for downloading the LLVM build from CI, and that doesn't happen since it doesn't see options from the default file included with `profile = "compiler"`

Reply:
I'm guessing this didn't happen because none of the options set in the default included files were relevant to `bootstrap.py` until now, only to the rust binaries ?